site/src/pages/release-notes/1.9.0.mdx

```diff
@@ -0,0 +1,237 @@
+  ```java
+  HttpRequest req = ...;
+  WebClient client = ...;
+  RequestOptions options = RequestOptions.builder()
+                                         .responseTimeout(Duration.ofSeconds(20))
+                                         .writeTimeout(Duration.ofSeconds(5))
+                                         .maxResponseLength(2048)
+                                         .attr(foo, "bar")
+                                         .build();
+  // Specify RequestOptions when sending a request.
+  client.execute(req, options);
+  
+  // Specify the options when building a request using WebClientRequestPreparation.
+  client.prepare()
+        .get("/some/path")
+        .responseTimeout(Duration.ofSeconds(20))
+        .writeTimeout(Duration.ofSeconds(5))
+        .maxResponseLength(2048)
+        .attr(foo, "bar")
+        .execute();
+  ```
+- You can now fluently build a <type://BlockingTaskExecutor> using <type://BlockingTaskExecutorBuilder>.
+  #3315 #3389
+  ```java
+  BlockingTaskExecutor.builder()
+                      .daemon(true)
+                      .numThreads(500)
+                      .threadNamePrefix("my-blocking-task-executor")
+                      .build();
+  ```
+- You can now easily get a boolean value from <type://QueryParams> and <type://HttpHeaders>. #3600
+  ```java
+  HttpHeaders headers = HttpHeaders.of("checked", "true", "valid", "1");
+  assert headers.getBoolean("checked");
+  assert headers.getBoolean("valid");
+  QueryParams params = QueryParams.of("checked", "false", "valid", "0");
+  assert !params.getBoolean("checked");
+  assert !params.getBoolean("valid");
+  ```
+- You can now conveniently configure a loopback port with
+  <type://ServerBuilder#localPort(int,Iterable)>. #3599 #3601
+  ```java
+  Server.builder()
+        .localPort(8080)
+        .localPort(9090, SessionProtocol.HTTP, SessionProtocol.HTTPS)
+  ```
+- You can now specify the default Jackson modules with <type://JacksonModuleProvider>. #3594 #3625
+  - All available Jackson modules are automatically loaded through Jackson's
+    [built-in SPI](https://fasterxml.github.io/jackson-databind/javadoc/2.11/com/fasterxml/jackson/databind/cfg/MapperBuilder.html#findAndAddModules--)
+    by default.
+- [TCP User Timeout Option](https://datatracker.ietf.org/doc/html/rfc5482) is enabled by default when an idle
+  timeout is enabled. #3509
+  - `TCP_KEEPIDLE` and `TCP_KEEPINTVL` options are enabled when a PING interval is greater than 0.
+- You can now get the last value of a header name or a query param using
+  <type://HttpHeaders#getLast(CharSequence)> and <type://QueryParams#getLast(String)>. #3438 #3568
+  ```java
+  HttpHeaders headers = ...;
+  headers.getLast(HttpHeaderNames.X_FORWARDED_FOR);
+  
+  QueryParams params = ...;
+  params.getLast("items");
+  ```
+- You can now specify the current and max content length when creating <type://ContentTooLargeException>.
```

Comment:
How about:
```suggestion
- You can now see the specific information such as the length of transferred, delta, and limit from <type://ContentTooLargeException>.
```
because there's not much chance that a user creates the exception by him/herself? 🤔

Reply:
That makes sense.